test/create.js

```diff
@@ -0,0 +1,50 @@
+var assert = require("assert");
+var Web3 = require("web3");
+var TestRPC = require("ethereumjs-testrpc");
+var Provider = require("../index");
+
+describe("Provider", function() {
+  var HttpProvider;
+
+  beforeEach("mock HttpProvider", function() {
+    HttpProvider = Web3.providers.HttpProvider;
+    Web3.providers.HttpProvider = TestRPC.provider;
```

Comment:
I'm a bit ambivalent on this because running an HTTP server is extra overhead... but it's probably actually what we want to test. Plus that means we could actually test for host/port hookup working, rather than just sending those options to the great /dev/null in the sky.

Will switch to that.